platforms/Windows/python/asserts/python.wxs

```diff
@@ -0,0 +1,11 @@
+<Wix xmlns="http://wixtoolset.org/schemas/v4/wxs">
+  <?define VariantName = asserts ?>
+  <?define VariantUpgradeCode = $(PythonUpgradeCode)?>
+  <?define VariantProductName = !(loc.EmbeddedPython_ProductName)?>
+  <?define VariantCabinetName = python.asserts.cab?>
```

Comment:
I agree with you, the cabinet name should not have `.asserts` in the name. We do not build the python interpreter with assertions.